src/xdk-simple-deploy.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  Sheet,
+  Workbook,
+  TextField,
+  Message,
+  LinkedField,
+} from '@flatfile/configure'
+
+// const LinkedSheet = new Sheet(
+//   'LinkedSheet',
+//   {
+//     firstName: TextField('First Name'),
+//     middleName: TextField('Middle'),
+//   },
+//   {}
+// )
+
+const TestSheet = new Sheet(
+  'TestSheet',
+  {
+    firstName: TextField('First Name'),
+    middleName: TextField('Middle'),
+    // linkedField: LinkedField({
+    //   label: 'Link to LinkedSheet',
+    //   sheet: LinkedSheet,
+    // }),
```

Comment:
Remove commented out code

Reply:
Or add linkedField back in if it's working

Reply:
Removed until we fully implement `LinkedField`